xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -764,21 +757,6 @@ CDateTime CPVRTimerInfoTag::EndAsLocalTime(void) const
   return retVal;
 }
 
-bool CPVRTimerInfoTag::IsEndAtAnyTime(void) const
-{
-  time_t time = 0;
-  CDateTime stop(m_StopTime);
-  stop.GetAsTime(time);
-  return time == 0;
-}
-
-void CPVRTimerInfoTag::SetEndAtAnyTime(void)
-{
-  time_t time = 0;
-  CDateTime stop(time);
-  SetEndFromUTC(stop);
-}
-
```

Comment:
Every data member of the tag must be serialized. Please, just "copy" from the stuff already present.